src/libpanic_unwind/seh.rs

```diff
@@ -164,9 +161,19 @@ pub struct _PMD {
 pub struct _TypeDescriptor {
     pub pVFTable: *const u8,
     pub spare: *mut u8,
+    #[cfg(bootstrap)]
```

Comment:
I'm actually sort of surprised that the `bootstrap` stuff is needed here, since this crate both defines catches and throws of panics.

Did this come up during testing though and it didn't pass without the flags? If it works it works and that's fine, just curious!

Reply:
Oh this may also be connected to lang item switcharoos which definitely would necessitate `#[cfg(bootstrap)]`

Reply:
Yes, it's because I've changed the exception type from `unsigned __int64*` to `struct rust_panic` (i.e. I've removed one layer of indirection). This matches up with the changes in `codegen_msvc_try`.